proto/server.proto

```diff
@@ -0,0 +1,15 @@
+syntax = "proto3";
+package server;
+
+option go_package = "./;server";
+
+message PublishStreamRequest {
+  string channel = 1;
```

Comment:
should we split it up with scope/namespace/path?  or just use channel :thinking:

Reply:
I'd go with a channel - seems simpler to operate with one value